src/matchers/toHaveBeenCalledAfter/index.js

```diff
@@ -18,8 +20,24 @@ const failMessage = (firstInvocationCallOrder, secondInvocationCallOrder) => ()
   'Received second mock with invocationCallOrder:\n' +
   `  ${printReceived(secondInvocationCallOrder)}`;
 
+const mockCheckFailMessage = (value, isFirstValue) => () =>
+  matcherHint('.toHaveBeenCalledAfter') +
+  '\n\n' +
+  `Matcher error: ${isFirstValue ? 'first' : 'second'} value must be a mock or spy function\n` +
```

Comment:
See comment on `toHaveBeenCalledBefore` Matcher 😃

Reply:
Updated as in `toHaveBeenCalledBefore`